refurb/checks/readability/fluid_interface.py

```diff
@@ -0,0 +1,107 @@
+from dataclasses import dataclass
+
+from mypy.nodes import (
+    Block,
+    Statement,
+    AssignmentStmt,
+    MypyFile,
+    CallExpr,
+    MemberExpr,
+    NameExpr,
+)
+
+from refurb.checks.common import check_block_like
+from refurb.error import Error
+
+
+@dataclass
+class ErrorInfo(Error):
+    """
+    https://towardsdatascience.com/the-unreasonable-effectiveness-of-method-chaining-in-pandas-15c2109e3c69
+    Sometimes a return statement can be written more succinctly:
+
+    Bad:
+
+    ```
+    def get_tensors(device: str) -> torch.Tensor:
+        a = torch.ones(2, 1)
+        a = a.long()
+        a = a.to(device)
+        return a
+
+
+    def process(file_name: str):
+        common_columns = ["col1_renamed", "col2_renamed", "custom_col"]
+        df = spark.read.parquet(file_name)
+        df = df \
+            .withColumnRenamed('col1', 'col1_renamed') \
+            .withColumnRenamed('col2', 'col2_renamed')
+        df = df \
+            .select(common_columns) \
+            .withColumn('service_type', F.lit('green'))
+        return df
```

Comment:
The trailing `\`'s here are causing each line to be merged into one. This is what `refurb --explain FURB999` looks like:

````
```
def get_tensors(device: str) -> torch.Tensor:
    a = torch.ones(2, 1)
    a = a.long()
    a = a.to(device)
    return a


def process(file_name: str):
    common_columns = ["col1_renamed", "col2_renamed", "custom_col"]
    df = spark.read.parquet(file_name)
    df = df             .withColumnRenamed('col1', 'col1_renamed')             .withColumnRenamed('col2', 'col2_renamed')
    df = df             .select(common_columns)             .withColumn('service_type', F.lit('green'))
    return df
```
````

To fix this, replace `\` with `\\`, or turn this docstring into a raw docstring (ie, `r""" ... """`). This could probably be a separate check in Refurb/Ruff since it's kinda unexpected.